localisation/wc_religions.csv

```diff
@@ -252,9 +252,13 @@ CAPITALIST;Capitalist;Capitaliste;;;;;;;;;;;;x;
 pandaren_religion_group;Chi;chi;;;;;;;;;;;;x;
 lightbound;Lightbound;;;;;;;;;;;;;x;
 lightbound_DESC;A heretical offshoot of the naaru teachings, the adherents to the Lightbound doctrine are dedicated to spread the influence of the Light wherever they possibly can and convert the unbelievers by any means necessary, even forcibly.;;;;;;;;;;;;;x;
-shadowlander;Shadowlander;;;;;;;;;;;;;x;
-shadowlander_DESC;The Shadowlands are a dimension existing between worlds, where the souls of the mortal creatures go when they die. Their origins remain uncertain, but they have existed ever since mortal life first arose in the physical universe and seemingly span all worlds in existence. When a soul is called from the afterlife, it is free to roam the material realm if it is not restrained within moments of its arrival. Death knights, necromancers, and undead as a whole share a strong connection with the realm.;;;;;;;;;;;;;x;
-THE_FEL;the Fel;;;;;;;;;;;;;x;
+throsic;Throsic;;Throsisch;;;;;;;;;;;x;
+throsic_DESC;Many spirits of the fallen Drust have been banished into an offshoot plane within the Emerald Nightmare, that they later named Thros, the Blighted Lands, where they cultivated a potent form of death magic with bastardized druidic elements. This form of magic resulted from numerous attempts to bewitch the souls of fallen soldiers into stone constructs in order to use against them against human settlers. Following this, the Throsic form of necromancy prefers binding the souls of the dead into durable and unrelenting stone revenants or wicker golems, while the traditional form of corpse reanimation is much rarer. Its influence towars the wildlife and the living is usually perpetuated by the native covens. The most loyal of Throsian witches can also ascend into Matrons, an incorporeal banshee-like creature, through a ritualistic act of slitting their own throat in ultimate devotion to their horrific masters from beyond the veil.;;Viele Geister der gefallenen Drust wurden in einen Ableger des smaragdgr�nen Albtraums verbannt, den sie sp�ter Thros, die verseuchten Lande, nannten, wo sie eine m�chtige Form der Todesmagie mit verdorbenen druidischen Elementen erschufen. Diese Form der Magie entstand aus zahlreichen Versuchen, die Seelen gefallener Soldaten in steinerne Konstrukte zu bannen, um sie gegen menschliche Siedler einzusetzen. Demnach bindet die throsische Form der Nekromantie bevorzugt die Seelen der Toten in best�ndige und unbeugsame steinerne Wiederg�nger oder Weidengolems, w�hrend die traditionelle Form der Totenerweckung deutlich seltener praktiziert wird. Ihr Einfluss auf die Tierwelt und die Lebenden wird in der Regel durch die ans�ssigen Zirkel aufrechterhalten. Die treuesten unter den throsischen Hexen k�nnen auch zu Matronen aufsteigen, einer k�rperlosen, banshee-�hnlichen Kreatur, indem sie sich in einem rituellen Akt in �u�erster Hingabe an ihre schrecklichen Herren von jenseits des Schleiers die eigene Kehle durchschneiden.;;;;;;;;;;;x;
```

Comment:
I fear, this text might be too long in the info (F1) window.

Reply:
I currently have no way of testing this

Reply:
Shortened it.